webapp/src/js/enketo/widgets/mrdt.js

```diff
@@ -47,25 +47,39 @@ define( function( require, exports, module ) {
 
         if ( !service.enabled() ) {
             $translate( 'mrdt.disabled' ).then(function( label ) {
-                $el.append( '<p>' + label + '</p>' );
+                $el.find( '.or-appearance-mrdt-image' )
+                   .append( '<p>' + label + '</p>' );
             });
             return;
         }
 
         $el.on( 'click', '.btn.mrdt-verify', function() {
-            service.verify().then( function(image) {
+            service.verify().then( function(data) {
+                var image = data;
+                var timeTaken;
+                if (typeof(data) === 'object') {
```

Comment:
Under what condition would this not be an object?

Reply:
If they use the currently active wrapper with 3.6 then `data` will be a string which is then used to construct the data url

Reply:
But even with the current wrapper, the call to verify will return an object.

Reply:
You are right .. I mixed stuff .. will fix